GsaGH/Parameters/0_Model/GsaGridLine.cs

```diff
@@ -184,5 +187,29 @@ internal void UpdatePreview() {
         Circle = new Circle(plane, radius);
       }
     }
+
+    public GridLine GetApiGridLineToUnit(LengthUnit unit) {
+      GridLine gridLine = DuplicateApiObject();
+      gridLine.X = unit == LengthUnit.Meter ? GridLine.X : new Length(GridLine.X, unit).Value;
+      gridLine.Y = unit == LengthUnit.Meter ? GridLine.Y : new Length(GridLine.Y, unit).Value;
```

Comment:
Why to check meter?
 gridLine.Length = new Length(gridLine.Length, unit).Meters;

what about theta conversion?

Reply:
"Why to check meter?" - because we don't want to convert values if they are already in meter. we want to save time and memory.
"Also, why again to use original value through which duplicate api object has been created?" - because it's more readable to use original values to be converted and then set to duplicated object.